package.json

```diff
@@ -29,6 +28,8 @@
   },
   "devDependencies": {
     "@fullhuman/postcss-purgecss": "^2.1.2",
+    "@types/cheerio": "^0.22.22",
+    "@types/node": "^14.11.2",
```

Comment:
Would make errors on production because ts-node

Reply:
... and yes there is compiled index.js but that's only because of me being an idiot.